src/frontend/web_application/src/components/MessageList/index.js

```diff
@@ -1,7 +1,19 @@
 import { compose } from 'redux';
+import { createSelector } from 'reselect';
+import { connect } from 'react-redux';
 import { withTranslator } from '@gandi/react-translate';
 import Presenter from './presenter';
 
+const userSelector = state => state.user;
+
+const mapStateToProps = createSelector(
+  [userSelector],
+  usertState => ({
+    user: usertState.user,
```

Comment:
careful, userState might be undefined when starting the appllication (the request not yet fully loaded)

And an HoC can be useful : e.g withUser()